src/panels/lovelace/cards/hui-button-card.ts

```diff
@@ -186,12 +190,7 @@ export class HuiButtonCard extends LitElement implements LovelaceCard {
             `
           : ""}
         ${this._config.show_name
-          ? html`
-              <span tabindex="-1">
-                ${this._config.name ||
-                (stateObj ? computeStateName(stateObj) : "")}
-              </span>
-            `
+          ? html` <span tabindex="-1" .title=${name}> ${name} </span> `
```

Comment:
Please check the spaces here:
```suggestion
          ? html`<span tabindex="-1" .title=${name}>${name}</span>`
```